github/event_types.go

```diff
@@ -490,11 +490,11 @@ type PullRequestEvent struct {
 	PullRequest *PullRequest `json:"pull_request,omitempty"`
 
 	// The following fields are only populated by Webhook events.
-	Changes            *EditChange   `json:"changes,omitempty"`
-	RequestedReviewers []*User       `json:"requested_reviewers,omitempty"` // Populated in "review_requested", "review_request_removed" event deliveries.
-	Repo               *Repository   `json:"repository,omitempty"`
-	Sender             *User         `json:"sender,omitempty"`
-	Installation       *Installation `json:"installation,omitempty"`
+	Changes           *EditChange   `json:"changes,omitempty"`
+	RequestedReviewer *User         `json:"requested_reviewer,omitempty"` // Populated in "review_requested", "review_request_removed" event deliveries. A request for multiple users to review at once is split into multiple requests, each with a single, different RequestedReviewer.
```

Comment:
Note that in #824, we have both the singular and the plural forms.
Please see the discussion there.

Reply:
The discussion seems to state that there is only a `RequestedReviewer` field at the highest level of the `PullRequestEvent` object, signifying the particular user whose review has been requested at that instant, while there is a `RequestedReviewers` object in the `PullRequest` struct of the `PullRequestEvent` struct. This is reinforced by the JSON example above (which I copy and pasted from prod logs) and the JSON example in #824 .

There is a `PullRequestEvent.PullRequest.RequestedReviewers`, and a `PullRequest.RequestedReviewer`, but no `PullRequest.RequestedReviewers`

Reply:
@stephen-soltesz Am I right in interpreting your comments and the email from #824 ?

Reply:
@jusunglee your summary matches my understanding. 

Based on the observed JSON payloads and the message from support@, I believe that `PullRequestEvent.RequestedReviewers` was never correct. It matched the public docs, but now we know those docs are incorrect.

Reply:
@gmlewis Does this suffice? I noticed that #824 added both fields directly in the `PullRequestEvent` slot but I am confident that it was on accident

Reply:
OK, thank you, @jusunglee and @stephen-soltesz.